packages/wrangler/src/config/validation.ts

```diff
@@ -73,7 +73,8 @@ export function isPagesConfig(rawConfig: RawConfig): boolean {
 export function normalizeAndValidateConfig(
 	rawConfig: RawConfig,
 	configPath: string | undefined,
-	args: NormalizeAndValidateConfigArgs
+	args: NormalizeAndValidateConfigArgs,
+	parsedFormat: Config["parsedFormat"] = "toml"
```

Comment:
This function is internal, so there is no value in making this field optional, IMO. It is better to constrain all usage to provide this. That also helps to ensure we have considered every usage.

Also since this and `configPath` are similar properties (in their usage) I would propose to group this new param as the third one ahead of `args`.